src/components/dataset/query/sidebar/panels/InfoView.jsx

```diff
@@ -62,9 +95,43 @@ export class InfoView extends React.PureComponent {
   };
 
   render() {
-    const { classes, dataset } = this.props;
+    const { classes, dataset, user } = this.props;
     const { isAccordionExpanded, isHelpVisible, helpTitle, helpContent } = this.state;
 
+    const linkToBq = dataset.accessInformation?.bigQuery !== undefined;
+    const tables = dataset.schema.tables.map((table, i) => (
+      <li key={`${i}`} className={classes.listItem}>
+        {linkToBq && (
+          <TerraTooltip
+            title={`Click to navigate to the Google BigQuery console where you can perform more advanced queries against the ${table.name} dataset table`}
+          >
```

Comment:
I don't think there needs to be a tooltip for each table in the tables list in the InfoPanel. A ? icon that displays the tooltip after the "Tables" header would suffice, I think

Reply:
Also, I'm getting a TerraToolTip _and_ browser-generated tooltip when I hover

Reply:
Gah lemme see about that.  I guess I didn't mind the tooltip over individual tables and it ends up taking less space than having a new ? icon on there.  @lstrano-broad thoughts?